src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/profiler.css.twig

```diff
@@ -994,3 +994,60 @@ table.logs .metadata {
         margin-left: 2px;
     }
 }
+
+{# Help tooltip
+   ========================================================================= #}
+
+.help {
+    display: inline-block;
+    text-align: center;
+    background-color: #6da581;
```

Comment:
I propose to make the color a bit lighter (less contrasting with the background, e.g. a light grey). Help icons shouldn't get attention when looking the page, they should only be visible when one needs it.

I'm afraid using the highlight/primary color will make it too distracting.